R/glimpse.R

```diff
@@ -155,8 +159,7 @@ format_glimpse.list <- function(x, ..., .inner = FALSE) {
     return("list()")
   }
 
-  out <- map_chr(x, format_glimpse_1, .inner = TRUE)
-
```

Comment:
Nitpick: why did this line go away?

Reply:
If we call `format_glimpse_1()` here, we have the problem, that nested lists are being formatted recursively with the same function, which leads to the commas inside list elements of length > 1 also being colored in grey.
Therefore we created another function `format_glimpse_list()` that doesn't color the commas, which is now being called here.

We could also add a boolean argument to `format_glimpse_1()` to steer if we want the coloring or not.